.github/workflows/hub-api-cd.yml

```diff
@@ -8,6 +8,8 @@ on:
       - closed
     paths:
       - 'lambda_handlers/*.py'
+    paths-ignore:
```

Comment:
Both `paths` & `paths-ignore` cannot be in the same workflow - [link](https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-syntax-for-github-actions#excluding-paths). Anyway, `paths-ignore` is not needed, as we are only considering `*.py` in `paths`